app/models/pieces/pawn.rb

```diff
@@ -1,2 +1,27 @@
 class Pawn < Piece
+  def valid_move?(destination_row, destination_column)
+    return false if (destination_column == column_coordinate) && ((destination_row - row_coordinate).abs == 1) &&
+                    square_taken?(destination_row, destination_column)
```

Comment:
Everything else about this PR looks great. Go ahead & merge it yourself once you've made the changes. 😄